sw/device/tests/BUILD

```diff
@@ -3032,6 +3032,51 @@ opentitan_test(
     ],
 )
 
+_RV_DM_LC_DISABLED_JTAG_LC_STATES = get_lc_items(
+    CONST.LCV.TEST_UNLOCKED0,
+    CONST.LCV.TEST_LOCKED0,
+    CONST.LCV.DEV,
+    CONST.LCV.PROD,
+    CONST.LCV.PROD_END,
+    CONST.LCV.RMA,
+)
+
+_LC_STATES_DEBUG_DISALLOWED = [
+    CONST.LCV.TEST_LOCKED0,
+    CONST.LCV.PROD,
+    CONST.LCV.PROD_END,
+]
+
+[
+    opentitan_test(
+        name = "rv_dm_lc_disabled_jtag_{}".format(lc_state),
+        srcs = ["example_test_from_flash.c"],
+        cw310 = cw310_jtag_params(
+            bitstream = "//hw/bitstream:rom_with_fake_keys_otp_{}".format(lc_state),
```

Comment:
we are now using `otp = ` and let the exec_env provide the bitstream. You can use one of the standard otp configuration, .e.g [this BUILD file](https://github.com/lowRISC/opentitan/blob/master/sw/device/silicon_creator/rom/e2e/boot_policy_valid/BUILD) See #20389 or #20389.